routes/engineRouter.js

```diff
@@ -11,4 +11,8 @@ router.get("/about", (req, res) => {
     res.sendFile(path.join(__dirname, "../", "public", "views", "about.html"));
 });
 
+router.get("/welcome", (req, res) => {
+    res.sendFile(path.join(__dirname, "../", "public", "views", "welcome/index.html"));
```

Comment:
Change this as below
```suggestion
    res.sendFile(path.join(__dirname, "../", "public", "views", "welcome", "index.html"));
```